REPOSITORIES.md

```diff
@@ -0,0 +1,67 @@
+# AsyncAPI Repositories
+
+AsyncAPI is composed of several different repositories hosted on Github. This document describes them.
+
+Any repositories that you find under https://github.com/asyncapi that are not listed here are archived or deprecated.
+
+## [asyncapi][asyncapi]
+
+This is the main project. It contains the AsyncAPI specification, documentation, examples, and miscellaneous other files such as the [contributer's guide][contributers].
+
+## [asyncapi.github.io][web-source]
+
+This is the source code for the public [web site][web-site]. It is powered by [Hugo][hugo].
+
+## [asyncapi-node][asyncapi-node]
+
+This provides all the versions of the AsyncAPI schema.
+
+## [asyncapi-react][asyncapi-react]
+
+This is a React component that can render and validate your AsyncAPI specification file as documentation.
+
+## [asyncapi-server][asyncapi-server]
+
+This is a Javascript server application that uses AsyncAPI.
+
+## [editor][editor]
+
+This is the source for the online editor hosted [here][online-editor]. It’s a Docker/node application.
+
+## [extensions-catalog][extensions-catalog]
+
+This is a placeholder for definitions of AsyncAPI specification extensions and protocolInfo objects. Extensions allow you to extend the syntax of the AsyncAPI schema.
+
+## [generator][generator]
+
+This is an npm module that generates files from asyncapi specification files based on templates. It currently has templates for Java/Spring, markdown and html.
+
+## [parser][parser]
+
+This is a Golang program that parses AsyncAPI specification files. It compiles to a shared library that can be bound to programs written in other languages.
+
+## [parser-generator][parser-generator]
+
+This is an npm module that parses AsyncAPI specification files.
```

Comment:
The parser-generator is an experimental repo that generates all the Go models from the JSON Schema definition. Eventually, this will get merged into the parser repo, so I'd just remove it from this list.

Sorry that I haven't realized before.